goldstone/core/models.py

```diff
@@ -140,27 +159,27 @@ def outgoing_edges(cls):      # pylint: disable=R0201
 
         return []
 
-    @staticmethod
-    def identity(thing):
+    @classmethod
+    def identity(cls, thing):
```

Comment:
I'm not sure that 'identity' accurately describes the purpose of this method.  I spent 5 minutes or so trying to compare it to the mathematical identity function :-) From what I see in docstring and implementation, it seems to be either using one or more attributes from cloud data, and possibly existing model data to create a native_id value that should be unique across clouds (by using our own cloud ID if necessary).  I think that's all good if I understand it properly.  In that case, I'd just suggest a more descriptive method name like create_native_id.